web/packages/teleterm/src/logger.ts

```diff
@@ -26,15 +26,15 @@ export default class Logger {
   // Logger.init has already been called
   constructor(private context = '') {}
 
-  warn(message: string, ...args: any[]) {
+  warn(message: any, ...args: any[]) {
```

Comment:
This more or less follows [the type of Node's console.log](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/aa7292823f873b69c4ae8fc8989f6cc58969384a/types/node/console.d.ts#L221).